vllm/worker/model_runner.py

```diff
@@ -503,6 +515,13 @@ def _compute_lens(self, inter_data: InterDataForSeqGroup, seq_idx: int,
             inter_data.input_positions[seq_idx].extend(
                 range(context_len, seq_len))
 
+        if (seq_len - context_len) == 1:
+            inter_data.num_orig_input_tokens_list[seq_idx].append(
+                seq_data.get_prompt_len())
+        else:
+            inter_data.num_orig_input_tokens_list[seq_idx].extend(
```

Comment:
Can you explain / comment why the `context_len` matters here?

Reply:
Same as above (to keep the shape same as the `input_positions` tensor)